kong/clustering/services/sync/rpc.lua

```diff
@@ -185,22 +202,22 @@ local function do_sync()
 
   local ns_deltas, err = kong.rpc:call("control_plane", "kong.sync.v2.get_delta", msg)
   if not ns_deltas then
-    ngx_log(ngx_ERR, "sync get_delta error: ", err)
-    return true
+    return error_handler("sync get_delta error: " .. err)
```

Comment:
It changed the behavior, now `error_handler()` returns `nil`, not the original `true`?
Could you verify it again?

Reply:
fixed